Makefile

```diff
@@ -236,4 +232,7 @@ elasticsearch-operator-install:
 # uninstalls the elasticsearch operator
 elasticsearch-operator-uninstall:
 	olm_deploy/scripts/operator-uninstall.sh
-
+gen-dockerfiles:
+	./hack/generate-dockerfile-from-midstream > Dockerfile && \
```

Comment:
this seems like its missing the `Dockerfile.in` parameter

Reply:
It assumes defaults, which can be overriden